labs/sysdev-kernel-cross-compiling/sysdev-kernel-cross-compiling.tex

```diff
@@ -164,10 +173,13 @@ \section{Cross compiling}
 \code{make -j<n>} if you have multiple cores on your machine!
 
 Look at the kernel build output to see which file contains
-the kernel image.
+the kernel image (\code{zImage}).
 
+\ifdefstring{\labboard}{beaglebone}
+{Also look in the Device Tree Source directory (\code{arch/arm/boot/dts}) to see which \code{.dtb}
```

Comment:
Why would we want to make this beaglebone specific ? Here as well, we expect people to have followed the slides, and then be able to relate by themselves "Device Tree Source directory" with arch/arm/boot/dts, or arch/arm64/boot/dts.